ajax/libs/angular-pageslide-directive/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "angular-pageslide-directive",
+  "filename": "angular-pageslide-directive.min.js",
+  "author": "Daniele Piccone <mail@danielepiccone.com>",
+  "description": "AngularJS sliding panel for serving additional content from off the page",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:dpiccone/ng-pageslide.git"
+  },
+  "keywords": [
+    "angular",
+    "pageslide",
+    "offcanvas",
+    "modal",
+    "pane"
+  ],
+  "version": "2.1.4",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/dpiccone/ng-pageslide.git",
```

Comment:
Same change as line 8, it is working for now as long as somebody claims my old username

Reply:
Thank you @danielepiccone 
Would you please tell us which url is good to be added? `git://github.com/danielepiccone/ng-pageslide.git` or `git://github.com/dpiccone/ng-pageslide.git`
Thank you.

Reply:
`git://github.com/danielepiccone/ng-pageslide.git` is the right, updated one

Reply:
@danielepiccone Thanks.
@sashberd Would you please help revise this url to `git://github.com/danielepiccone/ng-pageslide.git`
Thank you very much.

Reply:
Also, please help modify the body of 2nd commit msg from `...due to non existense on git` to `due to nonexistence on git repo`.
Then, everything looks good to me.